include/pros/rotation.h

```diff
@@ -0,0 +1,184 @@
+/**
+ * \file pros/rotation.h
+ *
+ * Contains prototypes for functions related to the VEX rotation sensor.
+ *
+ * Visit https://pros.cs.purdue.edu/v5/tutorials/topical/rotation.html to learn
+ * more.
+ *
+ * This file should not be modified by users, since it gets replaced whenever
+ * a kernel upgrade occurs.
+ *
+ * Copyright (c) 2017-2020, Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef _PROS_ROTATION_H_
+#define _PROS_ROTATION_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+namespace pros {
+namespace c {
+#endif
+
+/**
+ * Reset rotation sensor 
+ *
+ * Reset rotation sensor by multiplying the rotation reading by 
+ * -1 and changes the angle by -180 degrees only if the direction 
+ * was recently reversed
+ *
+ * This function uses the following values of errno when an error state is
+ * reached:
+ * ENXIO - The given value is not within the range of V5 ports (1-21).
+ * ENODEV - The port cannot be configured as an rotation sensor
+ *
+ * \param port
+ *        The V5 rotation sensor port number from 1-21
+ * \return 1 if the operation was successful or PROS_ERR if the operation
+ * failed, setting errno.
+ */
+int32_t rotation_reset(uint8_t port);
```

Comment:
Should we even expose this? What value does it offer?

Reply:
Maybe we should make it `rotation_flip` where it calls reset and then flips the values (with the 180 deg subtraction to angle) and sets the negative ? If not I'll just take it out, but yes I agree it doesn't add that much value other than for people who want to flip the angle or positive/negative value of the position.

Reply:
where are you even getting this description from? as far as i can tell this is not what the SDK documentation says this function does